cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -69,25 +69,27 @@ To prune the invalid list manually, follow these steps:
 #. The final ``pruneTime`` is the minimum ``pruneTime`` across all replicated clusters (if
    any); let this be time ``t``.
 
-Now, the invalid transaction list can be safely pruned
-until ``(t - 1 day)`` using :ref:`a call <http-restful-api-transactions-truncate>`
-with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactions-truncate>`.
+Now, the invalid transaction list can be safely pruned until ``(t - 1 day)`` using a call
+to :ref:`truncate invalid transactions before a specific time
+<http-restful-api-transactions-truncate>` with the :ref:`Transaction Service HTTP RESTful
```

Comment:
Now, the invalid transaction list can be safely pruned until ``(t - 1 day)`` by making a REST call to :ref:`truncate invalid transactions before a specific time <http-restful-api-transactions-truncate>`.

I don't think referencing the Transaction Service HTTP RESTful API documentation is needed, what do you say?